website/src/components/Header/Header.stories.jsx

```diff
@@ -0,0 +1,25 @@
+import { SessionContext } from 'next-auth/react';
+import React from 'react';
+
+import { Header } from './Header';
+
+export default {
+  title: 'Example/Header',
+  component: Header,
+  parameters: {
+    // More on Story layout: https://storybook.js.org/docs/react/configure/story-layout
```

Comment:
we can probably delete this in the code.  If people get confused we can add explanations in a README.